src/v/cluster/bootstrap_backend.cc

```diff
@@ -150,6 +150,15 @@ bootstrap_backend::apply(bootstrap_cluster_cmd cmd) {
     _members_manager.local().apply_initial_node_uuid_map(
       cmd.value.node_ids_by_uuid);
 
+    // Apply cluster version to feature table: this activates features without
+    // waiting for feature_manager to come up.
+    if (cmd.value.founding_version != invalid_version) {
+        co_await _feature_table.invoke_on_all(
+          [v = cmd.value.founding_version](features::feature_table& ft) {
+              ft.set_active_version(v);
```

Comment:
I think we need to avoid doing this if the version is lower than the current active version, e.g. if we've already loaded a feature table snapshot and are now replaying the controller log that was started on an older version.

Reply:
Good catch, you're right.  I think we can be even stricter, and just only set the active version if the current active version is invalid_version: this should only be happening when we have a totally default-constructed feature table.